core/src/main/resources/IAF_Util/ConfigurationFileSystem.xml

```diff
@@ -0,0 +1,106 @@
+<module>	
+	<adapter name="FileSystem" description="Upload files to server" active="${fileSystem.active}">	
+		<receiver name="FileSystem">
+			<listener name="FileSystem"
+				className="nl.nn.adapterframework.http.rest.ApiListener"
+				uriPattern="fileSystem" 
+				method="POST"
+				multipartBodyName="destination"
+				authenticationMethod="AUTHROLE"
+				authenticationRoles="IbisDataAdmin"/>
+		</receiver>
+		<receiver name="FileSystem-jl">
+			<listener className="nl.nn.adapterframework.receivers.JavaListener" name="FileSystem-jl" serviceName="${fileSystem-jl.serviceName}" />
+		</receiver>
+
+		<pipeline firstPipe="switchCleanFilesystem">
+			<exits>
+				<exit state="success" path="EXIT" code="200"/>
+				<exit state="error" path="ERROR" code="500"/>
+			</exits>
+
+			<pipe name="switchCleanFilesystem" 
+				className="nl.nn.adapterframework.pipes.XmlSwitch"
+				sessionKey="cleanFilesystem" 
+				notFoundForwardName="uploadFiles">
+				<forward name="clean" path="deleteFilesystem" />
+			</pipe>
+
+			<pipe name="deleteFilesystem"
+				className="nl.nn.adapterframework.pipes.CleanupOldFilesPipe"
+				filePatternSessionKey="destination" 
+				subdirectories="true"
+				deleteEmptySubdirectories="true" 
+				lastModifiedDelta="-1">
+				<forward name="success" path="uploadFiles" />
+			</pipe>
+
+			<pipe name="uploadFiles"
+				className="nl.nn.adapterframework.pipes.ForEachChildElementPipe"
+				getInputFromSessionKey="multipartAttachments"
+				elementXPathExpression="/parts/part[@type='file']">			
+				<sender className="nl.nn.adapterframework.senders.IbisLocalSender"
+					javaListener="FileSystem_Child">
+					<param name="unzipFile" sessionKey="unzipFile"/>
+					<param name="file" sessionKeyXPath="part/@name"/>
+					<param name="destination" sessionKey="destination"/>
+				</sender>
+				<forward name="success" path="EXIT"/>
+				<forward name="exception" path="ERROR"/>
+			</pipe>
+		</pipeline>
+	</adapter>
+
+	<adapter name="FileSystem_Child">
+		<receiver name="FileSystem_Child" >
+			<listener name="FileSystem_Child" className="nl.nn.adapterframework.receivers.JavaListener"/>
+		</receiver>
+
+		<pipeline firstPipe="isZipfile">
```

Comment:
It is not necessary to set the firstPipe attribute, if it refers to the first pipe of the pipeline